### PR TITLE
Add in Go

### PR DESCRIPTION
**Pull request by andymckay**
Similar to Ruby #1, this adds in Go and uses the new syntax and category.

### CODE REVIEW

**Review by damccorm — APPROVED** [review]
LGTM

**Review by Myoazwan95 — COMMENTED** [review]
Duplicate of #1

**Comment by thedincbey** [comment]
Hello

File: ci/go.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    steps:
+
+    - name: Set up Go 1.12
+      uses: actions/setup-go@master
+      with:
+        version: 1.12
+      id: go
+
+    - name: Check out code into the Go module directory
+      checkout: true
```
[the remark this answers is not in the document]
Comment (Tom12343321) [reply ci/go.yml]: Badoo

**Review by NameNoQuality — APPROVED** [review]

**Pull request merged**